dbt/models/ccao/schema.yml

```diff
@@ -70,6 +70,12 @@ sources:
         tags:
           - type_condo
 
+      - name: python_model_dependency
```

Comment:
I recognize the name of this model is pretty weird, so I'm open to other suggestions!